src/Gitonomy/Git/Repository.php

```diff
@@ -521,23 +521,23 @@ public function setDescription($description)
      *
      * @return Repository the current repository
      */
-    public function pull()
+    public function pull(): bool
```

Comment:
Please don't add return types. We don't use these in the repo.

Reply:
Because we need to support PHP 5.6.

Reply:
hmm, right, my bad. Why is support for 5,6 actually needed? because atm anything below 7.2 isn't officially supported afaik

Reply:
Because some of the other code that depends on this package even within this organisation doesn't work on PHP 7.

Reply:
We've already dropped PHP 5.5 and below recently, but we feel we should keep PHP 5.6 for now. There are not any features in PHP 7 we actually need.